dataplane/linux/bpf_ep_mgr.go

```diff
@@ -288,6 +292,22 @@ func (m *bpfEndpointManager) OnUpdate(msg interface{}) {
 	}
 }
 
+func (m *bpfEndpointManager) onInterfaceAddrsUpdate(update *ifaceAddrsUpdate) {
+	if update.Addrs != nil && update.Addrs.Len() > 0 {
```

Comment:
What if the len drops to zero?

Reply:
Length dropping to zero is equivalent to IP deletion? In that case, make the interfaces dirty?

Reply:
Yes, it means all the IPs are gone (or the interface itself is gone)